src/puppetlabs/puppetdb/http/command.clj

```diff
@@ -1,13 +1,45 @@
 (ns puppetlabs.puppetdb.http.command
   (:require [puppetlabs.puppetdb.command.constants :refer [command-names]]
             [puppetlabs.trapperkeeper.core :refer [defservice]]
+            [clojure.string :as str]
```

Comment:
Did we decide on what the standard for this namespace should be? AFAIK Clojure core used `string`